src/Pos/ExecMode.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskell     #-}
+{-# LANGUAGE TypeFamilies        #-}
+
+module Pos.ExecMode
```

Comment:
I am very sad that this module doesn't have any comments. Maybe that's because it's WIP.